olm/buildDigestMap.sh

```diff
@@ -39,33 +51,35 @@ IMAGE_LIST=$(yq -r '.spec.install.spec.deployments[].spec.template.spec.containe
 OPERATOR_IMAGE=$(yq -r '.spec.install.spec.deployments[].spec.template.spec.containers[].image' "${CSV}")
 
 REGISTRY_LIST=$(yq -r '.spec.install.spec.deployments[].spec.template.spec.containers[].env[] | select(.name | test("IMAGE_default_.*_registry"; "g")) | .value' "${CSV}")
-REGISTRY_IMAGES=""
+REGISTRY_IMAGES_ALL=""
 for registry in ${REGISTRY_LIST}; do
-  extracted=$(${SCRIPTS_DIR}/dockerContainerExtract.sh ${registry} var/www/html/*/external_images.txt | tail -n 1)
+  # echo -n "[INFO] Pull container ${registry} ..."
+  ${PODMAN} pull ${registry} ${QUIET}
 
-  # Container quay.io/eclipse/che-devfile-registry:7.9.0 unpacked to /tmp/quay.io-eclipse-che-devfile-registry-7.9.0-1584588272
-  extracted=${extracted##* } # the last token in the above line is the path we want
-
-  echo -n "[INFO] Extract images from registry ${registry} ... "
-  if [[ -d ${extracted} ]]; then
-    # cat ${extracted}/var/www/html/*/external_images.txt
-    REGISTRY_IMAGES="${REGISTRY_IMAGES} $(cat ${extracted}/var/www/html/*/external_images.txt)"
-  fi
-  echo "found $(cat ${extracted}/var/www/html/*/external_images.txt | wc -l)"
-  rm -fr ${extracted} 2>&1 >/dev/null
+  REGISTRY_IMAGES="$(${PODMAN} run --rm  --entrypoint /bin/sh  ${registry} -c "cat /var/www/html/*/external_images.txt")"
```

Comment:
this new approach is faster and simpler and doesn't require converting a container to a tarball to a file system just to get one file.